packages/mobile/src/identity/verification.ts

```diff
@@ -224,8 +224,12 @@ export function* doVerificationFlow() {
     return true
   } catch (error) {
     Logger.error(TAG, 'Error occured during verification flow', error)
-    yield put(setVerificationStatus(VerificationStatus.Failed))
-    yield put(showErrorOrFallback(error, ErrorMessages.VERIFICATION_FAILURE))
+    if (error.message === ErrorMessages.SALT_QUOTA_EXCEEDED) {
```

Comment:
Probably, better to be done as a switch... But I thought it's not worth it for only two right now.

Reply:
I like `if` better personally :)